packages/fxa-content-server/app/scripts/app.js

```diff
@@ -1,12 +1,18 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-import 'core-js'
+import 'core-js';
+import * as tracing from 'fxa-shared/tracing/browser-tracing';
 import ConfigLoader from './lib/config-loader';
 import AppStart from './lib/app-start';
 
 const configLoader = new ConfigLoader();
 configLoader.fetch().then((config) => {
+  if (config.tracing) {
+    const flowId = $ && $(document.body).data('flowId'); // eslint-disable-line no-undef
```

Comment:
Is this the flowId from the flow metrics?

Reply:
Yes. We wanted to add this so we could aggregate traces across a flow. If there is a better way to get this value, I'd love to know.